helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```
@@ -52,3 +52,16 @@ spec:
         team: atlas
         topic: observability
         cancel_if_outside_working_hours: "true"
+    - alert: PrometheusJobFailures
```

```suggestion
    - alert: PrometheusJobScrapingFailure
```

Is that name not more meaningful?

---

Maybe, but I also dislike longer names. Just a preference.

---

Why do you dislike longer names? Being able to know at a glance what this alert is makes everything easier

---

As an example: 
PrometheusJobFailures hints that a job (i.e. cronjob) named Prometheus fails to run or is failing, having the notion of scraping ensure we knowwhat this is about :)